task.cnf

```diff
@@ -90,4 +90,4 @@ long_query_time = 0
 # binlog_expire_logs_seconds	= 2592000
 max_binlog_size   = 100M
 # binlog_do_db		= include_database_name
-# binlog_ignore_db	= include_database_name
+# binlog_ignore_db	= include_database_name
```

Comment:
![image](https://github.com/user-attachments/assets/0ec23942-8d61-44eb-9a54-e40cc3186e06)
Add Eof(End-of-file), need to add a blank line at the end